Src/IronPython.Modules/re.cs

```diff
@@ -306,7 +306,7 @@ internal RE_Pattern(CodeContext/*!*/ context, object pattern)
                 return RE_Match.makeMatch(_re.Match(input, pos), this, input, pos, input.Length);
             }
 
-            public RE_Match match(object text, [DefaultParameterValue(0)]int pos, int endpos) {
+            public RE_Match match(object text, int pos=0, int endpos=int.MaxValue) {
```

Comment:
I'd just get rid of the default values here since we already have overloads to deal with those cases.